pages/aboutus/policy/index.js

```diff
@@ -7,15 +7,23 @@ function policy() {
         <AboutUsNavigation />
         <div className={classes.aboutUsSpace}>
             <div className={classes.aboutUsItem}>
-                <h1>Policy</h1>
-                <p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium,totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta
-                    sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia
-                    consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui
-                    dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora
-                    incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis
-                    nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur?
-                    Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur,
-                    vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?"</p>
+                <h1>Allmänt</h1>
+                <p>Det övergripande syftet med vår behandling av dina personuppgifter är att kunna administrera din tillgång till, och din användning av,
```

Comment:
Another nice way of dealing with text content is to use markdown. That makes it easier to edit and work on the text, without implementing a full CMS.

Here's some information from the official NEXT documentation:
https://nextjs.org/learn/basics/data-fetching/blog-data